sublime_haskell_common.py

```diff
@@ -245,10 +260,15 @@ def call_and_wait_with_input(command, input_string, wait = True, **popen_kwargs)
     # For the subprocess, extend the env PATH to include the 'add_to_PATH' setting.
     extended_env = get_extended_env()
 
+    # If the log level is log_error or greater, then allow the child's output
+    # and stderr to percolate back up to us via a pipe. Otherwise, send the
+    # output to the OS's null output device:
+    log_level = int(get_setting_async('log', log_info))
+    procoutput = subprocess.PIPE if wait or log_level >= log_error else subprocess.DEVNULL
```

Comment:
No problem.
We already using `PyV3 = version[0] == "3"` to write different code for python 2 and 3. Maybe we just can use `subprocess.DEVNULL` in python 3 and `os.devnull` in python 2? Or just `os.devnull` in both pythons?